client/src/components/ThankYou/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import * as ROUTES from "../../constants/routes";
+import { Link } from "react-router-dom";
+
+class ThankYou extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div>
+        <div>
+          <h1>Thank You </h1>
+          <h3>Would you like to go to?</h3>
```

Comment:
Can you make this a `<p>` tag instead? What do you think about changing it to _Where would you like to go?_ instead?

Reply:
Or maybe even "Where to?"

Reply:
Done